invenio_app_ils/ui/backoffice/src/pages/ItemDetails/state/types.js

```diff
@@ -0,0 +1,6 @@
+export const SET_ITEM_FETCH_LOADING = 'SET_ITEM_FETCH_LOADING';
```

Comment:
Missing one `SET_ITEM_FETCH_ERROR`, no?
I would remove the `SET_` part in general...